Header header
uint32 hand_id
geometry_msgs/Vector3 direction
geometry_msgs/Vector3 normal
geometry_msgs/Vector3 velocity
geometry_msgs/Vector3 palmpos
geometry_msgs/Vector3 ypr

